include/revng-c/mlir/Dialect/Clift/IR/CliftTypes.h

```diff
@@ -15,8 +15,80 @@
 #include "revng/Support/Assert.h"
 
 #include "revng-c/mlir/Dialect/Clift/IR/Clift.h"
+#include "revng-c/mlir/Dialect/Clift/IR/CliftAttributes.h"
 #include "revng-c/mlir/Dialect/Clift/IR/CliftInterfaces.h"
 
 // This include should stay here for correct build procedure
 #define GET_TYPEDEF_CLASSES
 #include "revng-c/mlir/Dialect/Clift/IR/CliftOpsTypes.h.inc"
+
+namespace mlir::clift {
+
+struct ScalarTupleTypeStorage;
+
+/// Scalar tuples are used to represent the multiple register return values of
+/// raw function types. They are much like structs, but permit only scalar
+/// element types and have no explicit offsets for their elements.
+class ScalarTupleType : public Type::TypeBase<ScalarTupleType,
```

Comment:
Add a note about the fact that this is temporary, eventually we'll let RFTs return plain structs and get rid of this.
(this belongs to #1)